dsaT3/filplot_funcs.py

```diff
@@ -683,8 +671,17 @@ def filplot_entry(datestr,trigger_dict,
     else:
         showplot=True
 
-    ra_mjd, dec_mjd = dsautils.coordinates.get_pointing(ibeam, obstime=Time(timehr, format='mjd'))
-    l, b = dsautils.coordinates.get_galcoord(ra_mjd.value, dec_mjd.value)
+    # VR hack
+    try:
+        ra_mjd, dec_mjd = dsautils.coordinates.get_pointing(ibeam, obstime=Time(timehr, format='mjd'))
+        l, b = dsautils.coordinates.get_galcoord(ra_mjd.value, dec_mjd.value)
+    except:
+        ra_mjd = 1.0*u.deg
+        dec_mjd = 71.5*u.deg
+        l = 100.0
+        b = 50.0
+
+    
```

Comment:
Are there defaults we could use instead to make it obvious that this lookup has failed when looking at candplotter?